package.json

```diff
@@ -27,6 +26,7 @@
     "@libp2p/interfaces": "^3.0.3",
     "@libp2p/logger": "^2.0.0",
     "@libp2p/peer-id": "^1.1.15",
+    "@multiformats/multiaddr": "../js-multiaddr/",
```

Comment:
Watch out for this. It expects that repo to be there on your local drive, and on my branch.

TODO: 
* once https://github.com/multiformats/js-multiaddr/pull/262 has landed, change this to refer to that repo.
* once that change has made it into a release, turn this back into a normal version number requirement